### PR TITLE
docs/adr: Update March 2022 testnet ADR to clarify bridge node

### PR DESCRIPTION
**Pull request by renaynay**
Related to confusion in #548

### CODE REVIEW

File: docs/adr/adr-003-march2022-testnet.md
Comment (liamsi) [reply docs/adr/adr-003-march2022-testnet.md:30]: > ? I mean what is the difference between validator and "regular full node"?

see first and last node type here: https://docs.tendermint.com/master/nodes/#node-types

It exists and the main diff is that one actively signs and the other does not.

Comment (Wondertan) [reply docs/adr/adr-003-march2022-testnet.md:30]: Ok, so the confusion I had here is that full nodes do full validation and semantically validators, while validators do the same + block production, and their name doesn't semantically include this trait.

It's orthogonal to the PR, just clarifications. Resolve freely

**Review by adlerjohn — APPROVED** [review]

**Review by YazzyYaz — APPROVED** [review]
LGTM, once it's merged, I can update the definition in the docs for Bridge Nodes.

**Review by liamsi — APPROVED** [review]
LGTM. I'm wondering if we should again tag @musalbas on this in the context of https://github.com/celestiaorg/celestia-node/issues/548

**Pull request merged**
